hopdrops/hopdrops
# feat(locations): use live location data from Square

> Caveat: You need to use a CORS proxy in order for this to work on localhost due to security in the browser
> Available Proxy:
> "https://cors-anywhere.herokuapp.com/https://api.hopdrops.io/v1/square/locations'

## Test w/ Proxy:
1. Edit `/src/actions/square.js` -> update with proxy
```js
  try {
    const response = await window.fetch(
      'https://cors-anywhere.herokuapp.com/https://api.hopdrops.io/v1/square/locations',
      {
```
2. Go through the login flow to get a JWT from hopdrops.
3. Copy and paste the token into the locations page with prefix with a hash: `http://localhost:3000/settings/locations#eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJtZXJjaGFudF9pZCI6IlpSTlQwREdEM0IyUEMifQ.NRwrund9XG_3Z1lZMlPa8u8k0x8b1g9j_ioFOlpQVIA`
4. The locations should load in the table:
![image](https://user-images.githubusercontent.com/6020066/65570390-cc83ae80-df15-11e9-9179-40a2901a127e.png)
